<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UdpSettingsDialog</class>
 <widget class="QDialog" name="UdpSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UDP Bridge Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="connectionGroup">
     <property name="title">
      <string>Connection</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="ipLabel">
        <property name="text">
         <string>Target IP:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="ipEdit">
        <property name="text">
         <string>127.0.0.1</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="portLabel">
        <property name="text">
         <string>Port:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="portSpinBox">
        <property name="minimum">
         <number>1024</number>
        </property>
        <property name="maximum">
         <number>65535</number>
        </property>
        <property name="value">
         <number>4242</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="protocolLabel">
        <property name="text">
         <string>Protocol:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="protocolCombo">
        <item>
         <property name="text">
          <string>OpenTrack (6 doubles)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>FreeTrack</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QCheckBox" name="autoStartCheck">
        <property name="text">
         <string>Auto-start when tracking begins</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
     <widget class="QGroupBox" name="controlsGroup">
      <property name="title">
       <string>Manual Controls</string>
      </property>
      <layout class="QVBoxLayout" name="controlsVLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="startStopButton">
           <property name="text">
            <string>Start UDP Stack</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="statusLabel">
           <property name="text">
            <string>Status: Stopped</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="buttonLayout">
         <item>
          <widget class="QPushButton" name="recenterButton">
           <property name="text">
            <string>Recenter</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pauseButton">
           <property name="text">
            <string>Pause/Resume</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   <item>
    <widget class="QPushButton" name="installWineButton">
     <property name="text">
      <string>Install UDP Bridge (Wine/Proton)...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="hotkeysGroup">
     <property name="title">
      <string>Hotkeys (Wine/Proton)</string>
     </property>
     <layout class="QGridLayout" name="hotkeysLayout">
      <item row="0" column="0" colspan="2">
       <widget class="QCheckBox" name="enableHotkeysCheck">
        <property name="text">
         <string>Enable hotkey utility</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="recenterKeyLabel">
        <property name="text">
         <string>Recenter key:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QKeySequenceEdit" name="recenterKeyEdit">
        <property name="keySequence">
         <string>F12</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="pauseKeyLabel">
        <property name="text">
         <string>Pause key:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QKeySequenceEdit" name="pauseKeyEdit">
        <property name="keySequence">
         <string>Pause</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>UdpSettingsDialog</receiver>
   <slot>accept()</slot>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>UdpSettingsDialog</receiver>
   <slot>reject()</slot>
  </connection>
 </connections>
</ui>
